guidelines/releases/release-process.md

```diff
@@ -7,102 +7,183 @@ This document describes how to create a Kyma release using Prow.
+
+    To see all release jobs for the 0.6 release, look for job names with the `pre-rel06` prefix.
+    Since every job name has to be unique, prefix it with `pre-rel{XY}`.
+    Define the release jobs on the `master` branch since Prow reads the job configuration from the `master` branch of the `test-infra` repository.
 
 2. Ensure that tests for release jobs exist. Release tests usually iterate through all release versions and run tests for them.
 See the `TestBucReleases` test defined in `development/tools/jobs/kyma/binding_usage_controller_test.go` as a reference.
 To add tests for all jobs for the new release, update the `GetAllKymaReleaseBranches()` function
 defined in the `development/tools/jobs/tester/tester.go` file under the `test-infra` repository.
 
 3. Define branch protection rules for the release branch in the `prow/config.yaml` file.
-For example, see the release-0.6 definition:
-```
-release-0.6:
-  protect: true
-  required_status_checks:
-    contexts:
-      - pre-rel06-kyma-integration
-      - pre-rel06-kyma-gke-integration
-      - pre-rel06-kyma-artifacts
-      - pre-rel06-kyma-installer
-```
-
+    For example, see the release-0.6 definition:
+
+    ```yaml
+    release-0.6:
+      protect: true
+      required_status_checks:
+        contexts:
+          - pre-rel06-kyma-integration
+          - pre-rel06-kyma-gke-integration
+          - pre-rel06-kyma-artifacts
+          - pre-rel06-kyma-installer
+    ```
 
 ## Release
 
+> **IMPORTANT:** Never use `/test all` as it might run tests that you do not want to execute!
+>
+> **NOTE:** Currently it is most likely required to push to the release-X.X branch more than once. Make sure that someone with admin permissions on github.com/kyma-project/kyma is available  
+> Before every release you need to bump the version in github.com/kyma-project/test-infra `prow/RELEASE_NUMBER`. This requires the approval of a member of wg-prow!  
+
 Follow these steps to create a release:
+
+### kyma-project/test-infra
+
 1. Create a release branch in the `test-infra` repository. The name of this branch should follow the `release-x.y` pattern, such as `release-0.6`.
 
->**NOTE:** This point only applies to new major and minor versions.
+    ```bash
+    git fetch upstream
+    git checkout -b $RELEASE_NAME upstream/master
+    ```
+
+    >**NOTE:** This point only applies to new major and minor versions.
 
 2. Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on a release branch contains the correct version to be created.
 The file should contain a release version following the `{A}.{B}.{C}` or `{A}.{B}.{C}-rc{D}` format, where `A`,`B`, `C`, and `D` are numbers.
 If you define a release candidate version, a pre-release is created.
 
-3. Create a release branch in the `kyma` repository. Do it only for a new release, not for a bugfix release.
+    > **NOTE:** make sure the RELEASE_VERSION file has only one single line:  
```

Comment:
Let's move it out of the note, since the text becomes a bit crammed with notes.

```
Make sure the `RELEASE_VERSION` file includes just this single line:  

` echo -n $RELEASE_VERSION > prow/RELEASE_VERSION`
```